src/main/java/ch/uzh/ifi/hase/soprafs24/service/RatingService.java

```diff
@@ -61,8 +54,8 @@ public List<Rating> getRatingsOfAnUser(Long userId, String token){
         List<Rating> reviewList = this.ratingRepository.findRatingsByReviewedId(userId);
         return reviewList;
     } 
-
-
```

Comment:
findReviews has been commented out because unused. 
findCreatedJobs and findHelpedJobs do not belong here and exist in the respective services. Furthermore, in the findCreatedJobs and findHelpedJobs the helper input is never used. In the findHelpedJobs the taskRepository.findByHelperId is called passing creator.getId(), which is confusing.